### PR TITLE
SPV-876: Adjust Transaction Page

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: src/components/RecordTxDialog/RecordTxDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    try {
+      if (hexOrId.length === 64) {
+        const response: any = await fetch(`https://api.whatsonchain.com/v1/bsv/main/tx/${hexOrId}/hex`);
+        const hex = await response.text();
+        await spvWalletClient!.AdminRecordTransaction(hex);
+
+        toast.success('Recorded Transaction');
+        setHexOrId('');
+        return;
+      } else if (hexOrId.length > 64) {
```
Comment (chris-4chain) [inline src/components/RecordTxDialog/RecordTxDialog.tsx:47]: If you do "return" you don't have to use "else" below. 
Or - if you want to use "if..else", skip the return part

File: src/routes/(admin)/_admin.transactions.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      }),
+    );
+  },
+});
+
+export function Transactions() {
+  const { spvWalletClient } = useSpvWalletClient();
+  const [tab, setTab] = useState<string>('all');
+  const [blockHeight, setBlockHeight] = useState<string>('');
+  const [debouncedFilter] = useDebounce(blockHeight, 200);
```
Comment (chris-4chain) [inline src/routes/(admin)/_admin.transactions.tsx:56]: Maybe `debouncedBlockHeight` ?

File: src/routes/(admin)/_admin.transactions.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  const { order_by_field, sort_direction } = useSearch({ from: '/_admin/transactions' });
+
+  const { data } = useSuspenseQuery(
+    // At this point, spvWalletClient is defined; using non-null assertion.
+    transactionsQueryOptions({
+      spvWalletClient: spvWalletClient!,
+      blockHeight: +debouncedFilter,
```
Comment (chris-4chain) [inline src/routes/(admin)/_admin.transactions.tsx:63]: Why the `+` sign 🤔 ?

File: src/components/RecordTxDialog/RecordTxDialog.tsx (hunk 1 above)
Comment (Nazarii-4chain) [reply src/components/RecordTxDialog/RecordTxDialog.tsx:47]: You're right. Thanks

File: src/routes/(admin)/_admin.transactions.tsx (hunk 3 above)
Comment (Nazarii-4chain) [reply src/routes/(admin)/_admin.transactions.tsx:63]: To convert to number

Comment (Nazarii-4chain) [reply src/routes/(admin)/_admin.transactions.tsx]: Changed to `Number`

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**